src/sequentia/rpc/rawtransaction.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2010 Satoshi Nakamoto
+// Copyright (c) 2009-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <node/coin.h>
+#include <node/psbt.h>
+#include <rpc/blockchain.h>
+#include <rpc/rawtransaction_util.h>
+#include <rpc/server_util.h>
+#include <script/standard.h>
+#include <sequentia/rpc/rawtransaction.h>
+#include <validation.h>
+
+namespace sequentia
+{
+
+static std::vector<RPCArg> CreateTxDoc()
+{
```

Comment:
I don't get what that is and how that differs from the other ones. Good enough for now I suppose?

Reply:
But then make it a branch until it's somewhat cleaner?

Reply:
It's a helper function for generating RPC documentation specific to `createrawtransaction`. I haven't changed it at all from the Elements source (yet).

Reply:
> But then make it a branch until it's somewhat cleaner?

Do you mean adding a runtime condition to the original `rawtransaction.cpp`?

Reply:
Oh, maybe you were asking about the entire module.

> I don't get what that is and how that differs from the other ones. Good enough for now I suppose?

It's effectively a way to override static methods by importing namespace methods from a different module.

> But then make it a branch until it's somewhat cleaner?

I don't think #ifdefs containing more than a few lines of code would be cleaner, and in this case it'd be a few dozen. And we're going to want to override both `createrawtransaction` and `CreateTxDoc` soon..